docs/reference/storage-paths.md

```
@@ -73,4 +72,4 @@ codex auth list
 
 - [../configuration.md](../configuration.md)
 - [../upgrade.md](../upgrade.md)
-- [../privacy.md](../privacy.md)
+- [../privacy.md](../privacy.md)
```

markdownlint-cli2 (0.21.0)

[warning] Files should end with a single newline character

(MD047, single-trailing-newline)